openshift/system/Dockerfile

```diff
@@ -91,11 +82,23 @@ RUN export ${BUNDLER_ENV} >/dev/null \
     && rm log/*.log \
     && chmod g+w /opt/system/config
 
-USER 1001
 ADD openshift/system/entrypoint.sh /opt/system/entrypoint.sh
+USER 1001
 EXPOSE 3000 9306
 # TODO: dumb-init!
 ENTRYPOINT ["/opt/system/entrypoint.sh"]
 CMD ["unicorn", "-c", "config/unicorn.rb", "-E", "${RAILS_ENV}", "config.ru"]
 
-# vim: set ft=dockerfile:
+
+FROM porta-base AS porta-sphinx-config
+USER 0
+RUN yum install -y mariadb-server \
+    && setpriv --reuid 27 --regid 27 --clear-groups mysql_install_db \
+    && (mysqld_safe &) \
+    && DATABASE_URL='mysql2://root:@localhost/3scale_system_production' SECRET_KEY_BASE=dummy RAILS_ENV=production /opt/system/entrypoint.sh bundle exec rake db:create db:schema:load openshift:thinking_sphinx:configure \
```

Comment:
Is this a disposable database? Why does Sphinx need a mysql database to generate the config?

Reply:
sphinx doesn't generate the config. Porta does and to start porta, one needs a database. Not sue what  you mean by "disposable" database. Image layer is gone afterwards so state of DB doesn't matter.